test/terraform_aws_nextjs_serverless_test.go

```diff
@@ -0,0 +1,52 @@
+package test
+
+import (
+	"crypto/tls"
+	"fmt"
+	"strings"
+	"testing"
+
+	http_helper "github.com/gruntwork-io/terratest/modules/http-helper"
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	"github.com/stretchr/testify/assert"
+	"github.com/tidwall/gjson"
+)
+
+func TestTerraformCloudFrontServerlessNextJS(t *testing.T) {
```

Comment:
Why do we testing three things in the same test and describe with comments what we do instead of creating 3 different test cases?

Reply:
Updated the test scenarios.